example/src/pages/LinePage.tsx

```diff
@@ -65,19 +67,29 @@ const Line: React.FC<{}> = () => {
     const state: IPointSet[] = [{
         name: 'UP',
         points: [
-            {
-              x: 343,
-              y: 281
-            },
-            {
-              x: 898,
-              y: 389
-            }
+          {
+            x: 1048,
+            y: 456
+          },
+          {
+            x: 1613,
+            y: 584
+          },
+          {
+            x: 1322,
+            y: 985
+          },
+          {
+            x: 922,
+            y: 785
+          }
         ],
         showPointHandle: true,
         showSmallDirectionMark: true,
         readOnly: false,
-        styling: 'primary'
+        styling: 'primary',
+        areaFillColor: 'rgb(0,0,0)',
```

Comment:
Instead of rgb if we take color in hex then it will better.

Reply:
Done
![image](https://github.com/future-standard/scorer-ui-kit/assets/118800413/cca4309a-555d-495d-94c2-1d0d67a6c54b)